extensions/amp-animation/amp-animation.md

```diff
@@ -394,17 +202,15 @@ Another way to specify keyframes is in the document's stylesheet (`<style>` tag)
 </amp-animation>
 ```
 
-CSS `@keyframes` are mostly equivalent to inlining keyframes definition in the JSON per [Web Animations spec](https://www.w3.org/TR/web-animations/#processing-a-keyframes-argument). However, there are some nuances:
+Most CSS `@keyframes` match the JSON inline keyframes definition in the [Web Animations spec](https://www.w3.org/TR/web-animations/#processing-a-keyframes-argument) with the following nuances:
 
-- For broad-platform support, vendor prefixes, e.g. `@-ms-keyframes {}` or `-moz-transform` may be needed. Vendor prefixes are not needed and not allowed in the JSON format, but in CSS they could be necessary.
-- Platforms that do not support `calc()` and `var()` will not be able to take advantage of `amp-animation` polyfills when keyframes are specified in CSS. It's thus recommended to always include fallback values in CSS.
-- CSS extensions such as [`width()`, `height()`, `x()`, `y()`, `num()`, `rand()`, `index()` and `length()`](#css-extensions) cannot be used in CSS.
+- You may need vendor prefixes, such as `@-ms-keyframes {}` or `-moz-transform` for broad-platform support. Vendor prefixes are not needed and not allowed in the JSON format, but in CSS they could be necessary.
+- In unsupported platforms, `amp-animation`'s pollyfills will fail when using calc() and var() with keyframes specified in CSS. Use fallback values in CSS to avoid this.
+- CSS extensions such as `width()`, `height()`, `x()`, `y()`, `num()`, `rand()`, `index()` and `length()` are not available to keyframes.
 
-#### White listed properties for keyframes
+##### White listed properties for keyframes
```

Comment:
Not an expert on this, but imho it should be "Whitelisted". Or even better "Allowed".

Reply:
another great catch - thanks